api/user/delete.py

```
@@ -1,27 +1,32 @@
+router = APIRouter()
+
+# FUNCTION -> DECLARATION
+@router.post("/api/user/delete")
+async def output(request: Request, response: Response) -> JSONResponse:
+    # DECLARATION -> EXTENSIONS
+    from website.extensions import database
+    await asyncio.gather(
+        database.init(request, response)
+    )
+    # DECLARATION -> USER AUTHENTIFIED WITH PERMISSIONS
+    if not (database.validate and database.user["Urole"] >= SUG.PER["user"]["delete"]): raise SUG.ERR[2]
```

Ruff (0.11.9)

`SUG` may be undefined, or defined from star imports

(F405)